crates/aptos/src/move_tool/mod.rs

```diff
@@ -40,13 +52,100 @@ impl MoveTool {
     pub async fn execute(self) -> CliResult {
         match self {
             MoveTool::Compile(tool) => tool.execute_serialized().await,
+            MoveTool::Init(tool) => tool.execute_serialized_success().await,
             MoveTool::Publish(tool) => tool.execute_serialized().await,
             MoveTool::Run(tool) => tool.execute_serialized().await,
             MoveTool::Test(tool) => tool.execute_serialized().await,
         }
     }
 }
 
+/// Creates a new Move package at the given location
+#[derive(Parser)]
+pub struct InitPackage {
+    /// Name of the new move package
+    #[clap(long)]
+    name: String,
+    /// Path to create the new move package
+    #[clap(long, parse(from_os_str))]
+    package_dir: PathBuf,
+    /// Path to the Aptos Framework locally
+    #[clap(long, parse(from_os_str))]
+    framework_dir: PathBuf,
```

Comment:
Why it's required and why it's present at all? If we would want to pre-populate dependencies with aptos stdlib, isn't it easier to just use git deps?

Reply:
I had done the whole git dependency, but it did not play well with my setup (2FA).  So, I need extra time to dive into whatever the inners of Move is doing, and to ensure it supports it better.

Reply:
Ok, still, `--framework-dir` as a required parameter seems really confusing to me.
Let's at least make this parameter optional, if user specifies it, then local dependency will be created, otherwise none would be. 
Users would first need to clone aptos-core repo, which is not an obvious step, they may as well add the record themselves.

Reply:
I'll make it default for github, but we'll need to improve this later.